<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>356</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>356</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordField">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rememberPasswordBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>181</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remember password</string>
   </property>
  </widget>
  <widget class="QLabel" name="loginWithQRLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;
&lt;body&gt;
&lt;p  style=&quot;line-height:50%; padding-botton: 0px&quot; align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Login with QR code&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot;&gt;Use the Steam mobile app&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="qrParentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <widget class="QPushButton" name="regenerateQRButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate New Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="qrErrorLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;
&lt;body&gt;
&lt;p  style=&quot;line-height:50%; padding-botton: 0px&quot; align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;QR Expired&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot;&gt;Press button below to regenerate&lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="usernameField">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
